comp/core/tagger/tagger.go

```diff
@@ -298,6 +300,10 @@ func (t *TaggerClient) globalTagBuilder(cardinality collectors.TagCardinality, t
 		}
 	}
 	t.mux.RUnlock()
+
+	if err := t.defaultTagger.AccumulateTagsFor(collectors.HostEntityID, cardinality, tb); err != nil {
```

Comment:
The purpose of `collectors.GlobalEntityID` was equivalent to `collectors.HostEntityID`
So I'm wondering if instead of creating a new entity `HostEntityID` we should add the host tags to the `GlobalEntityID`.

Reply:
I removed `HostEntityID` and bundled the host tags in with `GlobalEntityID` under a different source.